migrate.md

```diff
@@ -41,16 +41,16 @@ If you created service instances before November 2018, then depending on the loc
 ## Migrating a service instance
 {: #migrate-task}
 
-If you want to learn more about the migration process before you begin, see the [{{site.data.keyword.cloud_notm}} migration documentation ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-migrate){: new_window}.
+If you want to learn more about the migration process before you begin, see the [{{site.data.keyword.cloud_notm}} migration documentation](/docs/resources?topic=resources-migrate){: external}.
 
-Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more information, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: new_window}.
+Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more details, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: external}.
```

Comment:
Oh, my, changing "for more information" to "for more details"?  Unbelievable.

Reply:
Not sure where that came from. Perhaps it's a regression. Will reverse.